templates/404.html

```diff
@@ -0,0 +1,10 @@
+<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left"}} -->
+<main class="wp-block-group">
+    <!-- wp:pattern {"slug":"twentytwentyfive/hidden-404"} /-->
+</main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","area":"footer","tagName":"footer"} /-->
+ 
```

Comment:
Please change this file to using tabs, not spaces, and remove the trailing space at the end.
The last line of the file should be empty.
```suggestion

```